src/NuGet.Clients/NuGet.Credentials/PluginCredentialProvider.cs

```diff
@@ -124,11 +125,9 @@ public PluginCredentialProvider(Common.ILogger logger, string path, int timeoutS
 
                 if (response.IsValid)
                 {
-                    ICredentials result = new NetworkCredential(response.Username, response.Password);
-                    if (response.AuthTypes != null)
-                    {
-                        result = new AuthTypeFilteredCredentials(result, response.AuthTypes);
-                    }
+                    var result = new AuthTypeFilteredCredentials(
```

Comment:
Why change this?

It has no functional impact as far as I can tell.

Reply:
It's not PluginCredentialProvider's responsibility to figure out what the filter means